### PR TITLE
basic bug fixes without breaking backwards compatibility

### PR DESCRIPTION
**Pull request by rmartin16**
basic bug fixes without breaking backwards compatibility (except for Series_Images.summary() not settings attrs anymore...)

series.py
  Series_Episodes:
  - fixed _URLS{'queryparams'} to _URLS{'query_params'} so Series_Episodes.query_params() actually works
  - converted _PAGES, _PAGES_LIST, and _FILTERS from class members to instance members so calls to Series_Episodes works more than once
  Series_Images
  - fixed _URLS{'queryparams'} to _URLS{'query_params'} so Series_Images.query_params() actually works
  - stop settings attrs in summary

updates.py
  Updates:
  - fixed _URLS{'params'} init to _URLS{'update_params'} so Updates.update_params() actually works

search.py
  Search
  - fixed _URLS{'seriesparams'} to _URLS{'series_params'} so Search.series_params() actually works

user.py
  User_Ratings
  - converted _PAGES, _PAGES_LIST, and _FILTERS from class members to instance members so calls to User_Ratings works more than once

base.py
  TVDB
  - fixed 'error' key in to the JSON returned in the event of get_token or _request fail

### CODE REVIEW

**Comment by phate89** [comment]
Everything seems fine.. the only thing I'm missing (probably because I didn't look at this code since a lot of time) why you dropped setting attrs in summary..

**Comment by rmartin16** [comment]
Definitely feels like more than a month ago I posted this...:)...had to dig out the notes I made. Here's what I wrote about setting the attributes in Series_Images.

> series.py
> Series_Images
> - *call summary(), the other methods break since they have become integer attributes
>   - this does seem straight up broken since it seems most valuable to call this prior to calling the other methods
> - *once one of the other methods is called, that method becomes a attribute of the response and thus uncallable
>   - that said, you really only need to call these once per instantiation
> - It seems that summary should not be setting these attributes

Basically, calling Series_Images.summary() completely hoses your instance of Series_Images. So, not setting the attrs doesn't really break compatibility...it just makes summary() at least potentially useful to ever call.

File: tvdbsimple/series.py [hunk 1]
```diff
@@ -134,6 +131,10 @@ def __init__(self, id, language='', **kwargs):
         self._set_language(language)
         self.update_filters(**kwargs)
 
+        self._PAGES = -1
+        self._PAGES_LIST = {}
+        self._FILTERS = {}
```
Comment (phate89) [inline tvdbsimple/series.py]: You're overwriting possible filters set by update_filters... I'll merge https://github.com/phate89/tvdbsimple/pull/3/files that fixes the same problem and it's an older pr.. can you drop the change?

File: tvdbsimple/user.py [hunk 2]
```diff
@@ -165,6 +162,10 @@ def __init__(self, user, key, **kwargs):
         self._FILTERS = {}
         self.update_filters(**kwargs)
 
+        self._PAGES = -1
+        self._PAGES_LIST = {}
+        self._FILTERS = {}
```
Comment (phate89) [inline tvdbsimple/user.py]: Same here..but in this case _FILTERS is already set so you could simply drop this line..

**Review by phate89 — COMMENTED** [review]
Thanks for the fixes.. I agree on summary.. if you Can fix these 2 small changes I'll finally (after a loong time) merge your pr

File: tvdbsimple/series.py (hunk 1 above)
Comment (rmartin16) [reply tvdbsimple/series.py]: good catch. done.

**Pull request merged**